extensions/mssql/src/features.ts

```diff
@@ -35,6 +35,8 @@ export class AgentServicesFeature extends SqlOpsFeature<undefined> {
 		contracts.AgentJobActionRequest.type
 	];
 
+	private onUpdatedHandlers: any[] = [];
```

Comment:
this actually doesn't need to be an array, since there should only ever be one handler on the extensionhost side (multiple listeners in main thread is still possible).  I'll switch to a scalar variable.